ansible/roles/ocp4-workload-mlops/tasks/install-argocd-operator.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
There is already a role for this. Please use it if possible.

https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles_ocp_workloads/ocp4_workload_argocd_operator